models/articles.js

```diff
@@ -1,14 +1,29 @@
+const { response } = require('../app');
 const db = require('../db/connection');
 
 exports.fetchArticlesById = (articleId) => {
     return db
     .query(
-        `SELECT * FROM articles WHERE article_id = $1`, [articleId]
+        `SELECT * FROM articles WHERE article_id = $1;`, [articleId]
     ).then((response) => {
         if(!response.rows.length)
         {
           return  Promise.reject({status: 404, msg: "Not Found"})
         }
         return response.rows[0]
     })
+}
+exports.updateArticlesById = (article, votes) => {
+    article.votes += votes
+    
+    return db.query(
+        `UPDATE articles SET votes = $1 WHERE article_id = $2;`, [article.votes, article.article_id]
+    ).then(() => {    
+    return db
+    .query(
+        `SELECT * FROM articles WHERE article_id = $1;`, [article.article_id]
```

Comment:
While this works fine you don't have to make this second query in order to get the updated article back from the database, have a think about how you would get this data from your initial query.